backend/song/tests/factories.py

```diff
@@ -1,57 +1,68 @@
 import factory
 
 from factory.django import DjangoModelFactory
-from backend.song import models
+from backend.song.models import (
+    Category,
+    Artist,
+    Album,
+    Song,
+    SongLink,
+    Lyric,
+    LyricRequest,
+    Language,
+    Translation
+)
 from backend.users.tests.factories import UserFactory
 
 class CategoryFactory(DjangoModelFactory):
     class Meta:
-        model = models.Category
+        model = Category
     category_name = factory.Faker('word')
 
 
-class AlbumFactory(DjangoModelFactory):
-    class Meta:
-        model = models.Album
-    album_name = factory.Faker('sentence', nb_words=5)
-
-
 class ArtistFactory(DjangoModelFactory):
     class Meta:
-        model = models.Artist
+        model = Artist
     first_name = factory.Faker('first_name')
     last_name = factory.Faker('last_name')
     is_still_alive = factory.Iterator([True, False])
 
 
+class AlbumFactory(DjangoModelFactory):
+    class Meta:
+        model = Album
+    album_name = factory.Faker('sentence', nb_words=5)
+    artist = factory.SubFactory(ArtistFactory)
+
+
 class SongFactory(DjangoModelFactory):
     class Meta:
-        model = models.Song
+        model = Song
     title = factory.Faker('sentence', nb_words=3)
+    slug = factory.Faker('slug', value=title)
     category = factory.SubFactory(CategoryFactory)
     album = factory.SubFactory(AlbumFactory)
     artist = factory.SubFactory(ArtistFactory)
-    slug = factory.Faker('slug', value=title)
 
 
 class SongLinkFactory(DjangoModelFactory):
     class Meta:
-        model = models.SongLink
+        model = SongLink
     provider = factory.Faker('word')
     link = factory.Faker('url')
     song = factory.SubFactory(SongFactory)
 
 
 class LanguageFactory(DjangoModelFactory):
     class Meta:
-        model = models.Language
+        model = Language
 
     language = factory.Faker('language_name')
```

Comment:
can we please use `text `here, the `language_name` is making the test failing in my mac